src/components/ChatEntry.js

```diff
@@ -2,21 +2,36 @@ import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
 
+const filledHeart = '❤️';
+const emptyHeart = '🤍';
+
 const ChatEntry = (props) => {
+  const heartOrNot = props.liked ? filledHeart : emptyHeart;
+  const className = props.sender === 'Estragon' ? 'remote' : 'local';
+
+  const handleClick = (id) => {
+    props.handleLike(id);
+  };
   return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+    <div className={`chat-entry ${className}`}>
+      <h2 className="entry-name"> {props.sender}</h2>
       <section className="entry-bubble">
-        <p>Replace with body of ChatEntry</p>
-        <p className="entry-time">Replace with TimeStamp component</p>
-        <button className="like">🤍</button>
+        <p>{props.body}</p>
+        <p className="entry-time">{props.timeStamp}</p>
```

Comment:
Here's where you'd use the provided `Timestamp` component in order to format this into a friendly-to-read date